### PR TITLE
:sparkles: Add the ability to translate annotation literals

### PR DESCRIPTION
**Pull request by abrugaro**
Solves https://github.com/konveyor/windup-shim/issues/129

This PR also:
* Adds an e2e test to cover the translation of a whole rule
* Replaces imports from `github.com/fabianvf/windup-rulesets-yaml` to `github.com/konveyor/windup-shim` as the first repo no longer exists.

### CODE REVIEW

File: data/test-rule.xml
Comment (jmle) [inline data/test-rule.xml:1]: Let's make both `test-rule.xml` and `test_rule.yaml` either with `-` or `_`

File: e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	if err != nil {
+		fmt.Println(err)
+	}
+	_, err = conversion.ConvertWindupRulesetsToAnalyzer(rulesets, "./data/test-ruleset.xml", "./test-rule-output", true, false)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer func() {
+		err := os.RemoveAll("./test-rule-output")
```
Comment (jmle) [inline e2e_test.go:28]: Let's add this to .gitignore just in case

File: main.go [hunk 2]
```diff
@@ -9,9 +9,9 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/fabianvf/windup-rulesets-yaml/pkg/conversion"
-	"github.com/fabianvf/windup-rulesets-yaml/pkg/execution"
-	"github.com/fabianvf/windup-rulesets-yaml/pkg/windup"
+	"github.com/konveyor/windup-shim/pkg/conversion"
+	"github.com/konveyor/windup-shim/pkg/execution"
+	"github.com/konveyor/windup-shim/pkg/windup"
```
Comment (jmle) [inline main.go:14]: :+1:

File: main.go [hunk 3]
```diff
@@ -43,7 +43,7 @@ func main() {
 			for _, location := range convertCmd.Args() {
 				rulesets := []windup.Ruleset{}
 				ruletests := []windup.Ruletest{}
-				err := filepath.WalkDir(location, walkXML(location, &rulesets, &ruletests, false))
+				err := filepath.WalkDir(location, WalkXML(location, &rulesets, &ruletests, false))
```
Comment (jmle) [inline main.go:46]: Do we need to export `walkXML`?

**Review by jmle — CHANGES_REQUESTED** [review]
:+1: looks good, just a couple of small things

File: data/test-rule.xml
Comment (abrugaro) [reply data/test-rule.xml:1]: Solved in 37d17f5631418c7547e5becc7b0f32e4aa7e6098

File: e2e_test.go (hunk 1 above)
Comment (abrugaro) [reply e2e_test.go:28]: Solved in 37d17f5631418c7547e5becc7b0f32e4aa7e6098

**Review by aufi — APPROVED** [review]
LGTM

**Pull request merged**